homeassistant/components/balboa/entity.py

```diff
@@ -43,3 +43,21 @@ class BalboaEntity(BalboaBaseEntity):
     async def async_added_to_hass(self) -> None:
         """Run when entity about to be added to hass."""
         self.async_on_remove(self._client.on(EVENT_UPDATE, self.async_write_ha_state))
+
+
+class BalboaControlEntity(BalboaBaseEntity):
```

Comment:
Why not inherit from `BalboaEntity`, then you don't need to replicate `async_added_to_hass`

Reply:
The BalboaControlEntity listens for events on the "**control**" specifically rather than on the "**client**", so that it is updated less frequently.

`self.async_on_remove(self._control.on(EVENT_UPDATE, self.async_write_ha_state))`
vs
`self.async_on_remove(self._client.on(EVENT_UPDATE, self.async_write_ha_state))`

Reply:
I see now

Maybe you should add a comment in the code as it looked identical shfn I looked originally.

Reply:
Done